homeassistant/components/bitvavo/sensor.py

```diff
@@ -0,0 +1,279 @@
+        super().__init__(coordinator, order)
+
+        self._name = "Bitvavo Orders - Open"
+        self._unique_id = "bitvavo_orders_open"
+
+    def _get_data(self):
+        """Return the data from self.coordinator.data."""
+        return self.coordinator.data[CONF_OPEN_ORDERS]
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return self._name
+
+    @property
+    def state(self):
+        """Return the state of the sensor."""
+        return len(self._get_data())
+
+    @property
+    def unique_id(self):
+        """Return a unique id for the sensor."""
+        return self._unique_id
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit the value is expressed in."""
+        return "Orders"
+
+
+class TotalAssetValue(CoordinatorEntity):
+    """Implementation of the total asset value sensor."""
+
+    def __init__(self, coordinator, currency):
+        """Initialize the sensor."""
+        super().__init__(coordinator)
+        self._currency = currency
+
+        self._name = f"Bitvavo Total Asset Value - {self._currency.upper()}"
+        self._unique_id = f"bitvavo_total_asset_value_{self._currency.lower()}"
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return self._name
+
+    @property
+    def state(self):
+        """Return the state of the sensor."""
+        total_base_asset_value = 0.0
+
+        for i in self.coordinator.data[CONF_BALANCES].keys():
+            try:
+                asset_value = self.coordinator.data[CONF_BALANCES][i][
+                    "asset_value_in_base_asset"
+                ]
+
+                total_base_asset_value += float(asset_value)
+            except KeyError:
+                continue
+
+        if self._currency == ASSET_VALUE_BASE:
+            total_asset_value = total_base_asset_value
+        else:
+            asset_value_pair_name = (self._currency + "-" + ASSET_VALUE_BASE).upper()
+            last_price = float(
+                self.coordinator.data[CONF_ASSET_TICKERS][asset_value_pair_name][
+                    "price"
+                ]
+            )
+
+            total_asset_value = total_base_asset_value / last_price
+
+        return round(total_asset_value, 4)
+
+    @property
+    def unique_id(self):
+        """Return a unique id for the sensor."""
+        return self._unique_id
+
+    @property
+    def unit_of_measurement(self):
+        """Return the unit the value is expressed in."""
+        return self._currency
+
+    @property
+    def icon(self):
+        """Icon to use in the frontend."""
+        return CURRENCY_ICONS.get(self.unit_of_measurement, DEFAULT_COIN_ICON)
+
+    @property
+    def device_state_attributes(self):
+        """Return additional sensor state attributes."""
+        return {
+            "note": f"Value is based on the last {self._currency}-{ASSET_VALUE_BASE} price of every coin in balance",
```

Comment:
Is this a state?

Reply:
No only a note about the sensor state, remove these altogether, or only leave attribution?
I read that extra_state_attributes() must not be used for static info, is device_state_attributes() still used for that?

Reply:
device_state_attributes == extra_state_attributes.
It has been renamed.

Reply:
Thanks, I renamed them all.